circe/src/main/scala/io/finch/circe/Encoders.scala

```diff
@@ -13,4 +13,9 @@ trait Encoders {
    */
   implicit def encodeCirce[A](implicit e: Encoder[A]): Encode.Json[A] =
     Encode.json((a, cs) => BufText(print(e(a)), cs))
+
+  implicit val encodeExceptionCirce: Encode.Json[Exception] =
```

Comment:
Fun fact: we don't have to provide Finch's `Encode.Json[Exception]` here. Instead, we can define Circe's `Encoder[Exception]` and it will be implicitly converted (the conversion is defined in this very file) into Finch's version when needed.